### PR TITLE
release notes for 0.12

### PR DESCRIPTION
**Pull request by grlee77**
Please review this draft of release notes for 0.12.

### CODE REVIEW

File: docs/release/0.12.0.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+* Peter Bell +
+* Gregory R. Lee
+* Stefan Peterson +
+* DWesl +
+
+A total of 4 people contributed PRs to this release.
+People with a "+" by their names contributed a patch for the first time.
+
+John Kirkham and Henry Gommersall also contributed by helping review PRs.
```
Comment (hgomersall) [inline docs/release/0.12.0.rst]: Very minor, but it's "Gomersall" (one m)

Comment (grlee77) [reply docs/release/0.12.0.rst]: Thanks, sorry about that!

**Comment by grlee77** [comment]
@DWesl, if you would like your real name to appear in the release notes instead, please provide it. Thanks

**Comment by DWesl** [comment]
@DWesl<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FDWesl&data=02%7C01%7Cdfw5129%40psu.edu%7C8356ad71fb444b8196a208d787b8fc7a%7C7cf48d453ddb4389a9c1c115526eb52e%7C0%7C0%7C637127100888076934&sdata=mHMMgQtkFoZg1jWqbxavyKd8ejj3Mf05E7b70HOXIjM%3D&reserved=0>, if you would like your real name to appear in the release notes instead, please provide it. Thanks

I'm fine with this as it is.

**Comment by grlee77** [comment]
@hgomersall, please approve/merge this one. thanks

**Pull request merged**